dm-backend/Models/ReturnRequest.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using System;
+using System.Data;
+using System.Data.Common;
+using System.Threading.Tasks;
+using MySql.Data.MySqlClient;
+using dm_backend.Models;
+using static dm_backend.Utilities.Readers;
+
+namespace dm_backend.Models
+{
+    public class ReturnRequestModel
```

Comment:
Can this be inherited from the Request model for reuse ?

Reply:
Yes but what about modelling?